packages/storybook/src/css-cardgroup.stories.tsx

```diff
@@ -0,0 +1,62 @@
+/* @license CC0-1.0 */
+
+import '@frameless/components-css/card/index.scss';
+import readme from '@frameless/components-css/card-group/README.md?raw';
+import { Card } from '@frameless/components-react/src/Card';
+import { CardGroup } from '@frameless/components-react/src/CardGroup';
+import type { Meta, StoryObj } from '@storybook/react';
+import { Paragraph } from '@utrecht/component-library-react';
+
+const meta = {
+  title: 'CSS Component/CardGroup',
+  id: 'css-card-group',
+  component: CardGroup,
+  argTypes: {
+    children: {
+      name: 'Content',
+      description: 'CardGroup',
+      type: {
+        name: 'string',
+        required: true,
+      },
+      defaultValue: '',
+    },
+  },
+  args: {
+    children: '',
+  },
+  tags: ['autodocs'],
+  parameters: {
+    docs: {
+      description: {
+        component: readme,
+      },
+    },
+  },
+} satisfies Meta<typeof CardGroup>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {
+    children: (
+      <>
+        <Card heading="Design system" headingLevel={2}>
+          <Paragraph>
+            Herbruikbare componenten onafhankelijk van huisstijl, daar mag je ons voor wakker maken! Frameless heeft al
+            aan meerdere white-label design systems mogen werken en is trots op onze bijdrage aan NL Design System.
+          </Paragraph>
+        </Card>
+        <Card heading="Design system" headingLevel={6}>
```

Comment:
Ik zou voor deze card dezelfde headingLevel gebruiken :) Ik denk namelijk dat we al testen dat de verschillende heading levels er hetzelfde uitzien al in de card stories.

Reply:
Dit heb ik gedaan omdat de styling van utrecht heading  4, 5 en 6 andere line height hadden en ik wou ze naast elkaar testen tot dat ik zo fix. En dit had ik gefikst🙂

Ik zal het weer veranderen naar headinglevel 2